azure-mgmt-search/src/main/java/com/microsoft/azure/management/search/CheckNameAvailabilityResult.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+package com.microsoft.azure.management.search;
+
+import com.microsoft.azure.management.apigeneration.Beta;
+import com.microsoft.azure.management.apigeneration.Fluent;
+import com.microsoft.azure.management.resources.fluentcore.model.HasInner;
+import com.microsoft.azure.management.search.implementation.CheckNameAvailabilityOutputInner;
+
+/**
+ * The result of checking for Search service name availability.
+ */
+@Fluent
+@Beta(Beta.SinceVersion.V1_2_0)
+public interface CheckNameAvailabilityResult extends HasInner<CheckNameAvailabilityOutputInner> {
+  /**
+   * @return a boolean value that indicates whether the name is available for
+   * you to use. If true, the name is available. If false, the name has
```

Comment:
@brjohnstmsft it comes from Swagger (https://github.com/Azure/azure-rest-api-specs/blob/master/arm-search/2015-02-28/swagger/search.json).